instrumentation/oppia_instrumentation_test.bzl

```
@@ -0,0 +1,40 @@
+"""
+Instrumentation macros to setup up end-to-end tests.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+def oppia_instrumentation_test(
+        name,
+        srcs,
+        deps):
+    """
+    Creates library, binary and a instrumentation test for each test suite.
+
+    Args:
+        name: str. The class name of the Test suite or the test file.
+        srcs: list of str. List of test files under instrumentation module.
```

```suggestion
        srcs: list of str. List of test files corresponding to this test suite.
```

We don't require these to be relative, so better to keep it generic to the rule.

---

Done